components/Navbar.js

```diff
@@ -280,7 +280,7 @@ const NAV_ITEMS = [
   },
   {
     label: "Projects",
-    href: "projects",
+    href: "/projects",
```

Comment:
Thanks for fixing this lol. I have clicked this nav and been brought to the wrong page so many times, but I was too lazy to fix it.